### ISSUE TITLE
1.20.4 Request: Carry On Support

### ISSUE THREAD

**Issue by LunaFabula** [Glitchfiend/BiomesOPlenty#2236]
### Overview

I'm still playing on 1.20.4....
Carry On is a mod that allows on to move entities around, such as Chest Boats - the problem is that when I tried to move a Redwood Chest Boat around using Carry On, my chest boat had items inside before I picked it up with Carry On - when I placed it back down, the chest boat inventory was deleted. I'm honestly not sure what exactly happened, but that was surprising - and annoying - to find out.

Basically, either support for Carry On or a fix for the Biomes o' Plenty chest boat inventories.

Additionally, could you fix the Chest Boat recipes to use the boats instead of planks (and backport/foreward-port that)?

### Why would this feature be useful?

For the Chest Boat **recipe**, less confusion on how to craft them.

For the Chest Boat **inventory**, I don't know.

**Comment by Forstride** [comment]
The inventories thing is already a known issue (#2025)

Also the recipe thing was already fixed at some point, either in 1.20.6 or 1.21.  And no it won't be backported.
